modules/owncloud_web/pages/web_for_admins.adoc

```diff
@@ -0,0 +1,11 @@
+= ownCloud Web for Admins
+
+:toc: right
+:toc-levels: 1
+
+== User Management
+
+== App Integration
+
+Applications like Microsoft Word run in a "frame" displayed on oCIS, meaning they are actually running on the host server in this example at Microsoft, not on oCIS. Alternatively you can install the respective apps from the ownCloud marketplace.
```

Comment:
I think we have to rephrase this a bit to:
* better distinguish between ocis and oc10
* when do we install an app and when do we use an iframe displaying a foreign running application etc